vmm/src/lib.rs

```diff
@@ -1866,6 +1867,7 @@ impl Vmm {
                     kvm_hyperv: vm_config.cpus.kvm_hyperv,
                     #[cfg(feature = "tdx")]
                     tdx_enabled: vm_config.is_tdx_enabled(),
+                    amx_enabled: vm_config.cpus.features.amx,
```

Comment:
Also may trigger deadlock.

Reply:
This one is fine as only one lock to `vm_config` is requested and acquired via `let vm_config = &src_vm_config.lock().unwrap();`.

Reply:
Right :)